java/sage/BigBrother.java

```diff
@@ -296,9 +297,12 @@ public static boolean isFullWatch(Watched w)
         (((double) w.getWatchDuration())/wa.duration) > COMPLETE_WATCH_FRACTION);
   }
 
+  public static long watchIgnoreTime = Sage.getLong("wizard/watch_ignore_time", WATCH_IGNORE_TIME);
```

Comment:
Can you move these up to the top of the file and make them 'static final long' and then get rid of the WATCH_IGNORE_TIME[_MOVIE] vars and just inline the default values into those getLong calls?

Reply:
Sure.